applications/notes/Notes.html

```diff
@@ -0,0 +1,28 @@
+<header>
+    Books to Read
+</header>
+<section>
+
+<body>
+    <hr>
+    <aside>
+    <ul>
+        <li>There There by Tommy Orange </li>
+    <li> Let's Go (So We Can Get Back) by Jeff Tweedy</li>
+    <li> World: Being Human in the Age of Algorithms by Hannah Fry</li>
+    <li> Can't Stop Won't Stop by Jeff Chang</li>
+<li> Less by Andrew Sean Greer</li>
+<li> Educated by Tara Westover</li>
+<li> Westover Grit by Angela Duckworth</li>
+<li> See You in the Cosmos by Jack Cheng</li>
+<li> Just kids by Patti Smith</li>
+<li> Arbritrary Stupid Goad by Tamara Shopsin </li>
```

Comment:
It might seem minor, but consistent formatting will make your HTML much easier for everyone to read and understand. Including you when you come back to change it later!